### PR TITLE
Add playbooks with supports deletion all currently supported rest-objects

### PR DESCRIPTION
**Pull request by Mityushin**

### CODE REVIEW

File: playbooks/rest_objects/nfsServer/delete.yml [hunk 1]
```diff
@@ -1,18 +1,19 @@
 ---
 - hosts: localhost
   vars_files:
-    - ../../group_vars/all
+    - ../../../group_vars/all
   tasks:
     - name: Get nfsServer
-      nfsServer:
+      commonGetPost:
```
Comment (marsofandrew) [inline playbooks/rest_objects/nfsServer/delete.yml]: I think, that someteimes it's better to use get requests from client  (REST object) modules, like:

    nfsServer:
      ...
      get:
         ...

**Review by marsofandrew — DISMISSED** [review]
It's OK, but why do you always use commonGetPost?

File: playbooks/rest_objects/nfsServer/delete.yml (hunk 1 above)
Comment (Mityushin) [reply playbooks/rest_objects/nfsServer/delete.yml]: What kind of sometimes are you talking about?

Comment (dadlex) [reply playbooks/rest_objects/nfsServer/delete.yml]: Name a reason for it. The sdk allows you to do that, and it should be mentioned in the docs, but both methods are equally good.

File: playbooks/rest_objects/nfsServer/delete.yml [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
           unityIP: "{{ unity_host }}"
           username: "{{ unity_username }}"
           password: "{{ unity_password }}"
         get:
+          resource_type: nfsServer
           fields: id
-      register: result1
-    - debug: var=result1['output']['get'][0]['id']
+      register: get_nfsServer_results
+    - debug: var=get_nfsServer_results['output']['get'][0]['id']
```
Comment (dadlex) [inline playbooks/rest_objects/nfsServer/delete.yml]: IMHO "_results" part in the variable is excessive when we ask for a single field. I think the proper form is "get_nfsServer_id"

File: playbooks/complex/demo_file_access.yml [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
+  vars_files:
+    - ../../group_vars/all
   tasks:
     - name: get disks
       virtualDisk:
        login:
-          unityIP: "{{ host }}"
-          username: "{{ username }}"
-          password: "{{ password }}"
+          unityIP: "{{ unity_host }}"
```
Comment (dadlex) [inline playbooks/complex/demo_file_access.yml]: As you mentioned: "In #41 and #44 we remove unity_host group var. Let's change it to unity_ip_source or unity_ip_dest"

**Review by dadlex — CHANGES_REQUESTED** [review]
As you mentioned: "In #41 and #44 we remove unity_host group var. Let's change it to unity_ip_source or unity_ip_dest"

**Review by dadlex — APPROVED** [review]

**Review by marsofandrew — APPROVED** [review]

**Pull request merged**